### PR TITLE
Issue 149_add player name

### PR DESCRIPTION
**Pull request by yuanboz**
Not completely finish this part but players can at least see their name that they've entered at the beginning at every page(not include accountEqn and accountEqnAct).

I put prompt function which will let the player enter the name in the constructor() since I tried to put it in the update() but the result is the same, always prompt users to enter their name before load the background. Still working on this part but I think it's not a big deal.

### CODE REVIEW

**Comment by yuanboz** [comment]
So I have added username to these two page but I still not figure out how to make prompt run in the update.
I don't know why if I wait until the prompt comes from the update, the username won't be updated and it will show the username I entered last time. So I leave that part in the constructor.

File: accountEqnAct.js [hunk 1]
```diff
@@ -707,6 +712,15 @@ class accountEqnAct extends Phaser.Scene {
 
   }
 
+    displayProfile() {
+        this.profile.alpha = 1.0;
+        this.userName = this.add.text(70,140, localStorage.getItem("playerName"), {
```
Comment (fenwickjb) [inline accountEqnAct.js:717]: Can we store the entire name like you are doing, but only display the first 8 characters? Long names spill into the room floor area. It looks like about 8 will fit "in the wall." 

Perhaps grab the full stored name and then pull out a substring of characters 0-7. Not sure how Javascript would do this, but there should be something.

Comment (yuanboz) [reply accountEqnAct.js:717]: Sure!

**Comment by yuanboz** [comment]
I think I have solved the length of username. So the new feature is user can only see the first 7 characters followed by “..." if the input are more than 7 characters.

**Pull request merged**